client/apps/shipping-label/state/reducer.js

```diff
@@ -208,6 +208,7 @@ reducers[ UPDATE_PACKAGE_WEIGHT ] = ( state, { packageId, value } ) => {
 	newPackages[ packageId ] = {
 		...newPackages[ packageId ],
 		weight: parseFloat( value ),
+		is_user_specified_weight: true,
```

Comment:
I don't disagree but is this a documented standard for keys? I see quite a few in snake case, but then again maybe there's an overriding reason for those like conforming to DB schema.